apps/hash-graph/lib/graph/src/store/postgres/query/entity_type.rs

```diff
@@ -100,7 +101,22 @@ impl PostgresQueryPath for EntityTypeQueryPath<'_> {
             })
             .chain(path.relations())
             .collect(),
-            _ => unreachable!("Invalid path: {self}"),
+            Self::EntityTypeEdge {
+                edge_kind:
+                    OntologyEdgeKind::ConstrainsValuesOn | OntologyEdgeKind::ConstrainsPropertiesOn,
+                path: _,
+                direction: _,
+                inheritance_depth: _,
+            }
+            | Self::PropertyTypeEdge {
+                edge_kind:
+                    OntologyEdgeKind::InheritsFrom
+                    | OntologyEdgeKind::ConstrainsLinksOn
+                    | OntologyEdgeKind::ConstrainsLinkDestinationsOn
+                    | OntologyEdgeKind::ConstrainsValuesOn,
+                path: _,
+                inheritance_depth: _,
+            } => unreachable!("Invalid path: {self}"),
```

Comment:
I avoided the blank `_` to catch newly added paths (here and in other locations)